master/getting-started/kubernetes/installation/hosted/hosted.md

```diff
@@ -15,7 +15,15 @@ kubectl apply -f calico.yaml
 
 > **NOTE**
 >
-> Make sure you configure the provided ConfigMap with the location of your etcd cluster before running the above command. 
+> Make sure you configure the provided ConfigMap with the location of your etcd cluster before running the above command.
+
+## RBAC
+
+If using the above manifest on an RBAC enabled cluster, apply the `ClusterRole` and `ClusterRoleBinding` specs as well:
```

Comment:
We do mention this on the index page of the hosted installation - so should we remove from there?

Also, does this need to be done before applying the calico.yaml?  If so, we should re-order.